src/form/Form.js

```diff
@@ -39,7 +39,7 @@
                     .name("select-test")
                     .label("Select Test")
                     .type("select")
-                    .selectOptions(["A","B","C"])
+                    .selectOptions([["AAA"],["BBB"],["CCC"]])
```

Comment:
this also can be done as this which is why we check if type is array below: [["AAA","text to display"],["BBB",text to display2"],["CCC",text to display3"]]